data/committees.js

```diff
@@ -161,14 +169,14 @@ export default [
 			// 	platform: 'website',
 			// 	link: 'https://design.uclaacm.com/',
 			// },
-			{
-				platform: 'facebook',
-				link: 'https://www.facebook.com/groups/acmdesign',
-			},
 			{
 				platform: 'instagram',
 				link: 'https://www.instagram.com/acmdesign.ucla',
```

Comment:
seems design's current insta link is outdated, here's the new one: https://www.instagram.com/ucladesign

Reply:
done